Frontend/src/pages/dashboard/FarmerDash.jsx

```
@@ -476,28 +360,28 @@ function OrdersDashboard() {
   const fetchCrops = async () => {
     try {
       setLoading(true);
-      await new Promise((resolve) => setTimeout(resolve, 1000)); // Simulate API call
-    } catch {
-      setError("Failed to fetch crops. Please try again.");
+      const response = await fetch("http://localhost:3000/api/crops"); // Direct backend URL
+      if (!response.ok) throw new Error(`HTTP error! Status: ${response.status}`);
+      const data = await response.json();
+      setCrops(data); // Update state with fetched data
+    } catch (error) {
+      setError("Failed to fetch crops: " + error.message);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)